resources/package.json

```diff
@@ -4,35 +4,38 @@
   "private": true,
   "homepage": "./",
   "dependencies": {
-    "@craco/craco": "^6.1.1",
+    "@geist-ui/react": "^2.1.5",
+    "@geist-ui/react-icons": "^1.0.1",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
+    "@testing-library/user-event": "^13.1.8",
     "@types/jest": "^26.0.20",
-    "@types/node": "^14.14.34",
+    "@types/node": "^15.0.2",
```

Comment:
[`@types/node@15.0.3`](https://www.npmjs.com/package/@types/node) was published yesterday, btw.

Reply:
You can't upgrade the `package-lock.json` file because we're using Yarn and a `yarn.lock` file which I'm updating 😄